pydantic/fields.py

```diff
@@ -894,13 +902,13 @@ class ComputedFieldInfo:
         decorator_repr: A class variable representing the decorator string, '@computed_field'.
         wrapped_property: The wrapped computed field property.
         return_type: The type of the computed field property's return value.
-        alias: The alias of the property to be used during encoding and decoding.
-        alias_priority: priority of the alias. This affects whether an alias generator is used
-        title: Title of the computed field as in OpenAPI document, should be a short summary.
-        description: Description of the computed field as in OpenAPI document.
-        examples: Example values of the computed field as in OpenAPI document.
+        alias: The alias of the property to be used during serialization.
+        alias_priority: The priority of the alias. This affects whether an alias generator is used.
```

Comment:
Should we talk about different levels here? Or leave that to the usage docs?

Reply:
🤷‍♂️ I think it's okay if we just leave it to usage docs, if someone wants to make a change they can, I certainly wouldn't object just didn't want to take a stand in this PR